File: src/main/java/de/storchp/opentracks/osmplugin/utils/TrackColorMode.java
package de.storchp.opentracks.osmplugin.utils;

public enum TrackColorMode {

    UNI(true),
    BY_TRACK(true),
    BY_SPEED(false);

    private final boolean supportsLiveTrack;

    public static final TrackColorMode DEFAULT = BY_TRACK;

    TrackColorMode(final boolean supportsLiveTrack) {
        this.supportsLiveTrack = supportsLiveTrack;
    }

    public boolean isSupportsLiveTrack() {
        return supportsLiveTrack;
    }
}

File: src/main/java/de/storchp/opentracks/osmplugin/utils/TrackColorMode.kt
package de.storchp.opentracks.osmplugin.utils

import de.storchp.opentracks.osmplugin.utils.TrackColorMode.BY_TRACK

val DEFAULT_TRACK_COLOR_MORE: TrackColorMode = BY_TRACK

enum class TrackColorMode(val supportsLiveTrack: Boolean) {
    UNI(true),
    BY_TRACK(true),
    BY_SPEED(false);
}
